material.angular.io/tools/audit-docs.js

```diff
@@ -52,7 +52,7 @@ const MIN_SCORES_PER_PAGE = [
   {
     url: '',
     minScores: {
-      pwa: 75,
+      pwa: 0,
```

Comment:
I disabled this completely because we haven't cared about being a PWA in years. The score dropped, because I removed some of the icons we weren't using.